scripts/initialize-celo.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+set -euo pipefail
+
+KEEP_CORE_PATH_DEFAULT=$(realpath -m $(dirname $0)/../../keep-core)
+CONFIG_DIR_PATH_DEFAULT=$(realpath -m $(dirname $0)/../configs)
+KEEP_ECDSA_PATH=$(realpath $(dirname $0)/../)
+KEEP_ECDSA_SOL_PATH=$(realpath $KEEP_ECDSA_PATH/solidity)
+
+# Defaults, can be overwritten by env variables/input parameters
+KEEP_CELO_PASSWORD=${KEEP_CELO_PASSWORD:-"password"}
+NETWORK_DEFAULT="local"
+CONTRACT_OWNER_CELO_ACCOUNT_PRIVATE_KEY=${CONTRACT_OWNER_CELO_ACCOUNT_PRIVATE_KEY:-""}
```

Comment:
Is it used anywhere? I see it is passed as `CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY` environment variable when executing `scripts/lcl-initialize.js` and `lcl-client-config.js` but those two places in `initialize-celo.sh` are the only occurrences of `CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY` in the entire `keep-ecdsa` I can find.

Reply:
Right, these scripts don't use it but if we want to run it against `Alfajores`, then we still need to set `CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY` when executing this script. That's why we need to have in our `help()` function.